packages/enzyme-context-apollo-3/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "enzyme-context-apollo-3",
+  "version": "1.0.0",
```

Comment:
Eeek sorry can you actually initialize this as `0.0.0`? I think that's gonna make it easier when it comes to publishing since we'll be prompted for the new version #, and at _that_ point we'll bump to 1.0.0